index.html

```diff
@@ -40,6 +41,188 @@ <h3 class="subject_text">I'm a Software Developer</h3>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+        <div class="project_container_6">
+            <div class="project_title">
+                <h3>Trading Website</h3>
+            </div>
+            <div class="project_description">
+                <h4>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. Has been the 
+                industry's standard.</h4>
+            </div>
+            <div class="project_image"></div>
+            <ul class="project_skills">
+                <li class="skill_container"><h4 class="skill">HTML</h4></li>
+                <li class="skill_container"><h4 class="skill">Bootstrap</h4></li>
+                <li class="skill_container"><h4 class="skill">Ruby</h4></li>
+            </ul>
+            <button class="project_link"><h4 class="link_text">See project</h4></button>
+        </div>
+    </section>
+    <section class="about_me">
+        <div class="bio_header">
+            <h1>About me</h1>
+        </div>
+        <div class="bio">
+            <h4>Hello I'm a software developer! I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you see and have a 
+            project you need coded, don't hesitate to contact me.</h4>
+        </div>
+
+        <a href="#" class="resume_link">
+            <button class="resume_button">
+                <h4 class="link_text">Get my resume</h4>
+            </button>
+        </a>
+        <div class="about_me_illustration"></div>
+        <div class="separation_line">
+            <hr>
+        </div>
+        <div class="knowledge">
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_1"></div>
+                    <h3 class="category">Language</h3>
+                </div>
+                <h4 class="competency">Javascript</h4>
+                <h4 class="competency">Ruby</h4>
+                <h4 class="competency">HTML</h4>
+                <h4 class="competency">CSS</h4>
+            </div>
+            <div class="competent">
+                <div class="category_style">
+                    <div class="bullet_2"></div>
+                    <h3 class="category">Frameworks</h3>
+                </div>
+                <h4 class="competency">React.js</h4>
+                <h4 class="competency">Ruby on Rails</h4>
+                <h4 class="competency">RSpec</h4>
+                <h4 class="competency">Capybara</h4>
+            </div>
```

Comment:
- Kindly, make sure you use the `ul` tag to list the elements on this block instead of using the `div` tag, and note that each should be inside a `li` tag.😄